src/services/alert/adapter/message.js

```diff
@@ -11,6 +11,7 @@ class Message {
 		this.mMessageId = messageId || '-1';
 		this.mUrl = url || '';
 		this.mStatus = status || '';
+		this.mCreatedAt = createdAt === undefined ? '' : createdAt;
```

Comment:
Wouldn't
```suggestion
		this.mCreatedAt = createdAt || '';
```
be sufficient here?